src/inputrules.ts

```diff
@@ -35,12 +36,17 @@ export class InputRule {
       /// When set to false,
       /// [`undoInputRule`](#inputrules.undoInputRule) doesn't work on
       /// this rule.
-      undoable?: boolean
+      undoable?: 'boolean',
```

Comment:
I don't think this should be a string.

Reply:
yes, you're right, don't know how I changed that.